tcms/core/tests/test_report_tag.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import unittest
+from tcms.core.templatetags.report_tags import percentage
+
+
+class TestReportTagPercentage(unittest.TestCase):
+
+    def test_percentage_calculation(self):
+
+        self.assertEqual(percentage(5.0, 2.5), '200.0%')
```

Comment:
this is functionally equivalent to `1 / 1 => 100%` below. Both of them have 0 decimal digits.
Remove this b/c you have a nice ordering going on below.